tools/DevicesApiTester/DeviceApiTester.csproj

```diff
@@ -12,7 +12,13 @@
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.3.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="2.10.0" />
-    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
+    <!-- Normally package reference should be used: -->
+    <!-- <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" /> -->
+    <!-- We use project reference in our repository -->
+    <!-- so that our build breaks immediately and not on the next day when the official build happens -->
+    <ProjectReference Include="..\..\src\System.Device.Gpio\System.Device.Gpio.csproj">
+      <AdditionalProperties>Configuration=Linux-Debug;RuntimeIdentifier=linux</AdditionalProperties>

```

Comment:
No need to set Configuration, RuntimeIdentifier should be good enough.

Reply:
I followed the pattern for existing projects - not sure why they did that but assuming it fixes some issue 😄 will do it though

Reply:
Because I wasn't getting it to work right in VS without it. YMMV.

Reply:
The Configuration part is only for VS if you had both projects loaded in the solution which you don’t in this case so that should not be needed

Reply:
actually seems like a build breaks when I remove this - it cannot find many files - I will take a look at this

Reply:
Actually - misread your comment and removed RuntimeIdentifier as well... why is this actually needed? can we do something that it is not?

Reply:
is it because you reference rid specific project from rid=any? Seems kinda weird that I have to pass that by hand